src/protections/MessageIsMedia.ts

```diff
@@ -35,7 +35,11 @@ export class MessageIsMedia extends Protection {
 
     public async handleEvent(mjolnir: Mjolnir, roomId: string, event: any): Promise<any> {
         if (event['type'] === 'm.room.message') {
-            const content = event['content'] || {};
+            let content = event['content'] || {};
+            const relation = content["m.relates_to"]
+            if (relation && relation["rel_type"] === "m.replace") {
```

Comment:
fwiw, this should resolve to the same thing:

```suggestion
            if (relation?.["rel_type"] === "m.replace") {
```